src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160526_index_workflow_status.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="index-workflow-status-1">
+        <sql stripComments="true">
+            alter table WORKFLOW modify STATUS varchar(32);
+        </sql>
+    </changeSet>
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="index-workflow-status-2">
+        <sql stripComments="true">
+            alter table WORKFLOW add index (status);
```

Comment:
now that we're avoiding lookups for status off of the table, we can index status allowing for queue status endpoint calls to avoid a full table scan and using all sorts of extra internal mysql functions (filesort, temporary tables)